pointerevents/pointerevent_styles.css

```diff
@@ -1,6 +1,6 @@
 #innerFrame {
 position: absolute;
-top: 250px;
+top: 300px;
```

Comment:
Note: In running the [current version of this test](http://w3c-test.org/pointerevents/pointerevent_attributes_mouse-manual.html) I see that these absolute styles make running on mobile viewports pretty difficult (the boxes overlap the test text).  It's a pre-existing issue, but related in that making this test be applicable for touch screens probably means it should also be applicable on phones.  I filed w3c/PointerEvents#139 to track this.